plugins/gpio/GPIODriver.cpp

```diff
@@ -48,6 +48,7 @@ using std::vector;
 
 GPIODriver::GPIODriver(const Options &options)
     : m_options(options),
+      m_first_slot(options.start_address - 1u),
```

Comment:
I don't believe you'll need the u after the 1 now. If you do, can you switch to casting it instead which is how we normally do this.